api/services/Reddit.js

```diff
@@ -1,38 +1,26 @@
-//
 var request = require("request"),
   moment = require('moment'),
   left = 600,
-  resetTime = moment().add(600, "seconds");
+  resetTime = moment().add(600, "seconds"),
+  userAgent = "Webpage:hq.porygon.co:v" + sails.config.version;
```

Comment:
I changed the user-agent string to be a bit more descriptive (it was hard-coded as `fapp/1.0` before). Now it updates with the version number. I used [these guidelines](https://github.com/reddit/reddit/wiki/API) to make this user-agent string, but it's up for modification.